src/main/java/seedu/manager/command/FilterCommand.java

```diff
@@ -39,8 +62,39 @@ public void execute() {
 
         switch (flag) {
         case "-e":
+            String lowerCaseKeyword = filterWord.toLowerCase();
+            filteredEvents = eventList.filterByKeyword(lowerCaseKeyword);
+            outputMessage.append(FILTER_BY_KEYWORD_MESSAGE + "\n");
+            break;
```

Comment:
I think you can extract out the code foe each case in this method for better SLAP?